app/src/main/java/com/lagradost/cloudstream3/ui/account/AccountSelectActivity.kt

```diff
@@ -51,9 +47,13 @@ class AccountSelectActivity : AppCompatActivity() {
         }
         recyclerView.adapter = adapter
 
-        recyclerView.layoutManager = if (isTvSettings()) {
-            LinearLayoutManager(this)
-        } else GridLayoutManager(this, 2)
+        if (isTvSettings()) {
+            val spanSize = if (accounts.count() <= 6) {
+                accounts.count()
+            } else 6
```

Comment:
I'm debating whether this should be 4 and the accounts UI scaled up a bit or 6

Reply:
I think 6 works fine, I might try something else later but this should be ok to merge for now